settings/security/third-party.rst

```diff
@@ -0,0 +1,50 @@
+Third-Party Applications
+************************
+
+Third party authentication is a great way to help your users to login to Zammad
```

Comment:
Third-party authentication is a great way to help your users to log in to Zammad
more easily